### PR TITLE
Added links to Red Hat Marketplace

### PR DESCRIPTION
**Pull request by ahardin-rh**
Per the request of Joe Schram and Chris Sessoms

FYI @vikram-redhat @adellape

### CODE REVIEW

File: modules/olm-why-use-operators.adoc [hunk 1]
```diff
@@ -21,11 +21,12 @@ needed to build complex distributed systems – secret handling, load balancing,
 service discovery, autoscaling – that work across on-premise and cloud
 providers.
 
-Why manage your app with Kubernetes APIs and `kubectl` tooling?::
-These APIs are feature rich, have clients for all platforms and plug into the
-cluster’s access control/auditing. An Operator uses the Kubernetes' extension
-mechanism, Custom Resource Definitions (CRDs), so your custom object, for
-example `MongoDB`, looks and acts just like the built-in, native Kubernetes
+Why manage your app with Kubernetes APIs and `kubectl` tooling?:: These APIs are
```
Comment (adellape) [inline modules/olm-why-use-operators.adoc:24]: Can you line break this after the `::`? It renders fine, just... OCD (to match the others). :smile:

**Review by adellape — COMMENTED** [review]
Looks good! Just the minorest of nits... <_<

File: modules/olm-why-use-operators.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/olm-why-use-operators.adoc]: Good catch!

**Pull request merged**